Modules/Core/Common/include/itkGaussianDerivativeOperator.h

```diff
@@ -81,7 +81,7 @@ extern ITKCommon_EXPORT std::ostream &
  * Primal Sketch. Dissertation. Royal Institute of Technology, Stockholm,
  * Sweden. May 1991.).
  *
- * \author Ivan Macia, VICOMTech, Spain, http://www.vicomtech.es
+ * \author Ivan Macia, VICOMTech, Spain, https://www.vicomtech.org/en
```

Comment:
Although at the time Ivan contributed with the class the company used the above case for its name, they currently use the case `Vicomtech`. If you are working on other changes within this PR, you may want to update this as well.

Reply:
@jhlegarreta Just to clarrify, do you mean changning it to `https://www.vicomtech.org/es/`?

Reply:
I think he meant `Ivan Macia, VICOMTech` should be `Ivan Macia, Vicomtech`.

Reply:
@dzenanz is right.